prow/gerrit/adapter/adapter.go

```diff
@@ -402,6 +416,11 @@ func (c *Controller) handleInRepoConfigError(err error, instance string, change
 
 // processChange creates new presubmit/postsubmit prowjobs base off the gerrit changes
 func (c *Controller) processChange(logger logrus.FieldLogger, instance string, change client.ChangeInfo) error {
+	tracker := time.Now()
+	defer func() {
+		logger.WithField("duration(s)", time.Since(tracker).Seconds()).Debug("Time taken for gerrit change to be processed.")
```

Comment:
nit: This sounds pretty similar to "Time taken for gerrit change to be processed by a worker thread.". I understand the distinction you are making, but I think it could be helpful to make the start times clearer. For example this measure the processing duration while the "Time taken for gerrit change to be processed by a worker thread." measures the latency since the changes were first queried to when the change is done being processed.

Reply:
added a comment to clarify a bit more